src/recipes.jl

```diff
@@ -913,6 +913,19 @@ end
 @deps pie shape
 
 
+# ---------------------------------------------------------------------------
+# mesh 3d replacement for non-plotly backends
+
+@recipe function f(::Type{Val{:mesh3d}}, x, y, z; i=nothing,j=nothing,k=nothing)
```

Comment:
this should also query `plotattributes[:connections]` instead of the keywords

Reply:
Oh yes, I forget that. I have changed it now.